clowder-framework/clowder
# Improve error-handling for geospatial previewer

## Description
If GeospatialViewer doesn't find any data to render, an empty tab appears alongside the other previewers.

To alleviate user confusion, hide this tab header and its contents if the data inside will be empty or blank.

Related to https://github.com/clowder-framework/clowder/issues/134

### How to Test
Prerequisites: enable RabbitMQ plugin

1. Start / register `image-preview` extractor
2. Upload a generic TIFF image file (e.g. non-geotiff)
3. Wait for extractor to finish generating a preview, then view the uploaded file in Clowder
    * You should see that only a Preview tab appears
    * You should see that the Preview tab is not empty, and contains a thumbnail of the TIFF image
    * You should **not** see a GeospatialViewer tab that is empty (note: if you used a geotiff you may see this tab, but in this case the tab should contain a working geo previewer)

## Review Time Estimate


## Types of changes


## Checklist:
